edit containment syntax

dvdherron opened the pull request:
![](https://source.unsplash.com/featured/?cute,animal&boop)
<!-- Swap `CHANGE_ME` for a random string to get a different image -->



## Description

Update syntax for creating containment context. 


## Steps to test/reproduce
See the Define a continment context section at `/2021/04/05/containerqueries/`
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._


## Show me
_Provide screenshots/animated gifs/videos if necessary._


# REMEMBER: Attach this PR to the Trello card

dvdherron commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Update container query syntax in Quick Start post](https://trello.com/c/axTqI8BA/143-update-container-query-syntax-in-quick-start-post)

mirisuzanne reviewed (commented):
Explanation here is great! I think we might also want to make it clear when/why this was updated?

stacyk commented on content/blog/2021/containerqueries.md:
```diff
@@ -63,34 +63,49 @@ container.
+and [`size`](https://drafts.csswg.org/css-contain/#size-containment)
+containment are applied automatically as well.
+
+(`Block-size` is also being considered as a containment type in the new
+proposal. `Inline-size` seemed to satisfy more use-cases, so is
```
would block-size and inline-size look weird if they were both lowercase?

dvdherron replied:
Hmm. I could try rewriting these sentences to keep those words from being at the beginning?

stacyk reviewed (approved):
This looks good to me. I think @mirisuzanne mentioned this but I agree it would be nice to know when something was changed and updated in this article so people who are looking at it know. I was looking at a Mozilla post and say it is outdated and can see why this stuff may get confusing as people start to learn it if things are not all up to date. A changelog for the blog post :)

dvdherron commented:
Added a changelog to the article @mirisuzanne @stacyk . Is the formatting/wording OK?

mirisuzanne reviewed (approved):
@dvdherron looks great!

I think this is fine to merge as-is, but it made me curious about adding a markdown-footnotes extension, so we can add footnotes like this and link them from the text? And or inline notices like you just added to css-sandbox. That could be a separate card?

dvdherron commented:
> 
> 
> @dvdherron looks great!
> 
> I think this is fine to merge as-is, but it made me curious about adding a markdown-footnotes extension, so we can add footnotes like this and link them from the text? And or inline notices like you just added to css-sandbox. That could be a separate card?

@mirisuzanne  Thanks! Looks like we already had the `markdown-it-footnote` plugin installed but I didn't find a place where were currently using it?

I cleaned this up with footnotes and updated the date. Also tried to make the text a little clearer if you want to give this one more look.

mirisuzanne reviewed (approved):
looks great!

stacyk commented:
Love the footnotes!

jgerigmeyer commented on content/blog/2021/containerqueries.md:
```diff
@@ -63,34 +63,54 @@ container.
+against the containing element's [inline
+axis](https://drafts.csswg.org/css-writing-modes-4/#inline-axis).
+[`Layout`](https://drafts.csswg.org/css-contain/#valdef-contain-layout) and
+[`size`](https://drafts.csswg.org/css-contain/#size-containment)^[_`Style` added
+as necessary value for `contain` property when establishing a containment
+context._ Updated **May 07, 2021**.] containment are applied automatically as
```
I don't really understand this second footnote. What was updated on May 7? It looks like `style` is now being removed in this update?

dvdherron replied:
@jgerigmeyer 

`size` was added as a required value of the old `contain` property on May 7. 

`contain: inline-size layout style;`

both `layout` and `style` are implied with the new `container-type` and `container-name` values, which replace `contain` in this context.

Tried to make that more clear:
![image](https://user-images.githubusercontent.com/41588129/125638354-6657c9a5-e8f2-47fd-8063-cb8448e4a7a7.png)

jgerigmeyer reviewed (approved):
LGTM 🚀
